README.md

```diff
@@ -34,12 +34,19 @@ If the installation of a package was not succesfull there will be information st
 
 
 ## This repo contains two notebooks :
+**These notebooks can be executed in a workflow using [papermill](https://github.com/nteract/papermill).**
+
 **1. Template notebook to pre-process solver dataset and output clean dataset which is the input for clustering : [notebooks/PreprocessSolverErrorData.ipynb](notebooks/PreprocessSolverErrorData.ipynb)** \
-The purpose of this notebook is to preprocess solver data, i.e, extract error data from solver data, prepare data for clustering, clean and tokenize the clustering data and save the clean dataset for ClusterError notebook.
-  
+The purpose of this notebook is to preprocess solver data, i.e, extract error data from solver data, prepare data for clustering, clean and tokenize the clustering data and save the clean dataset for ClusterError notebook.\
+Excuecute via CLI: \
+ `papermill PreprocessSolverErrorData.ipynb PreprocessSolverErrorData.ipynb -p get_fresh_data False`\
```

Comment:
the new notebook should have a new name I think

Reply:
Updated.